apis/logging/v1/index_management_types.go

```diff
@@ -49,6 +49,18 @@ type IndexManagementDeletePhaseSpec struct {
 	// The minimum age of an index before it should be deleted (e.g. 10d)
 	//
 	MinAge TimeUnit `json:"minAge"`
+	// +nullable
+	Namespace []DeleteNamespaceSpec `json:"deleteNamespace,omitempty"` //#BEE: Why is this an array?
+}
+
+type DeleteNamespaceSpec struct {
+	// The unique name of the spec
+	Name string `json:"name"` // delete-<namespace_name>
+	// Namespaces to be deleted
+	NamespacesToDelete []string `json:"namespace"`
+	// Delete the records matching the namespaces which are older
+	// than this MinAge (e.g. 1d)
+	MinAge TimeUnit `json:"minAge"`
```

Comment:
Besides that I suggest to improve naming a bit:
```go
type IndeManagementDeleteNamespaceSpec struct {
	// The unique name of the spec
	Name string `json:"name"` // delete-<namespace_name>

	// Namespace to delete documents from older than minAge.
	Namespace string `json:"namespace"`

	// Delete the records matching the namespaces which are older
	// than this MinAge (e.g. 1d)
	MinAge TimeUnit `json:"minAge"`
}
```

Consider the official kubernetes API conventions when adding/extending APIs:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md

Reply:
I can see that you changed the datatype of `Namespace` from `[]string` to` string`.  This is because this one:
`Namespaces []IndeManagementDeleteNamespaceSpec `json:"namespaces,omitempty"``
is already an array, right?

Reply:
Exactly each  namespace spec is dedicated to a single namespace.

Reply:
@periklis but the actual thought during proposal was that a user can have multiple namespace in a single spec.
For example, 
Spec-1:
Name - policy1
Namespace - openshift-api, openshift-logging
Age - 1d

Spec-2:
Name - policy2
Namespace - openshift-kube-server
Age - 5h